packages/socket/lib/browser.ts

```diff
@@ -5,3 +5,11 @@ export type { Socket } from 'socket.io-client'
 export {
   io as client,
 }
+
+export function createWebsocket ({ path, browserFamily }: { path: string, browserFamily: string}) {
+  return io({
+    path,
+    // TODO(webkit): the websocket socket.io transport is busted in WebKit, need polling
```

Comment:
What is the to-do around this? Wait for a new version of Webkit? Is there something special we need to do in Cypress to align these? or is this just a webkit thing and this comment could be updated to reflect that?

Reply:
There's not a TODO, But the comment should remain.

Reply:
```suggestion
    // the websocket socket.io transport is busted in WebKit, so need polling instead
```

Reply:
https://github.com/cypress-io/cypress/issues/23807 is the follow-up for this issue. It needs to be fixed before stable WebKit, the Network tab gets very noisy with polling.

```suggestion
    // TODO(webkit): the websocket socket.io transport is busted in WebKit, need polling
    // https://github.com/cypress-io/cypress/issues/23807
```

Reply:
updated